app/src/main/java/org/fossasia/openevent/events/NoInternetEvent.java

```diff
@@ -0,0 +1,7 @@
+package org.fossasia.openevent.events;
+
+/**
+ * Created by MananWason on 8/4/2015.
+ */
+public class NoInternetEvent {
```

Comment:
Retrofit provdes HTTP error codes for example 404 server not found 503 is
server crashed/too busy handling requests + timeout. You need to tell the
right error message. You just cannot tell the user, "Oh look your internet
is down" when infact it is alright.

On Mon, Aug 10, 2015 at 2:55 AM, Manan Wason <notifications@github.com>
wrote:

> In app/src/main/java/org/fossasia/openevent/events/NoInternetEvent.java
> <https://github.com/fossasia/open-event-android/pull/69#discussion_r36592247>
> :
>
> > @@ -0,0 +1,7 @@
> > +package org.fossasia.openevent.events;
> > +
> > +/**
> > + * Created by MananWason on 8/4/2015.
> > + */
> > +public class NoInternetEvent {
>
> I mean this was my thought behind doing this. There is no assumption. We
> can put good log messages and show the message as "trouble downloading
> data" or something like this. Thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fossasia/open-event-android/pull/69/files#r36592247>.
>



-- 
Mohit Singh KANWAL (Mr)

[image: Facebook] <http://facebook.com/mohitkanwal>  [image: Twitter]
<http://twitter.com/mohitkanwal>